docs/config.rst

```diff
@@ -315,7 +315,11 @@ A stack has the following keys:
   The base name for the stack (note: the namespace from the environment
   will be prepended to this)
 **class_path:**
-  The python class path to the Blueprint to be used.
+  The python class path to the Blueprint to be used. Specify this or
+  ``template_path`` for the stack.
+**template_path:**
```

Comment:
I kinda feel like this could all be done without even adding a new config option. The `RawTemplateBlueprint` could just use a variable to specify the template path.

For example:

```yaml
stacks:
- name: vpc
  class_path: stacker.blueprints.RawTemplateBlueprint
  variables:
    template_path: templates/vpc.yaml
```

Whether or not that's better is probably debatable, but it keeps stacker config's more generic, which is good.

Reply:
I think it's worth of the pain of class_path vs template_path to avoid any other config values on where the template is.